pyfar/dsp/dsp.py

```diff
@@ -1600,9 +1602,10 @@ def convolve(signal1, signal2, mode='full', method='overlap_add'):
     Returns
     -------
     signal : Signal
-        The result of the convolution. The
-        :py:mod:`cdim <pyfar._concepts.audio_classes>` matches the bigger cdim
-        of the two input signals.
+        The result of the convolution. The channel dimension (`cdim`) matches
+        the bigger cdim of the two input signals. The cdim is the length of the
+        channel shape (`cshape`), which gives the shape of the data inside an
+        audio object but ignores the number of samples or frequency bins.
```

Comment:
```suggestion
        the bigger cdim of the two input signals. The channel dimension gives the number of dimensions of the audio data excluding the last dimension, which is ``n_samples`` for time domain objects and ``n_bins`` for frequency domain objects.
```
adopted from https://pyfar.readthedocs.io/en/stable/classes/pyfar.audio.html#pyfar.classes.audio.FrequencyData.cshape